Makefile

```diff
@@ -47,6 +47,9 @@ XARGS := xargs -L 1
 
 include make/testing_flags.mk
 
+DEV_TAGS := $(if ${tags},$(TEST_TAGS) ${tags},$(TEST_TAGS))
```

Comment:
If `TEST_TAGS` are used for more than tests, should they maybe be named `DEV_TAGS` to begin with?

Reply:
yes good point :)

Reply:
done